diff --git a/ios/EXDevLauncherController.m b/ios/EXDevLauncherController.m
index 76fd0f5f15f2ee6beadca5ce3c9828f0427c8896..806afb683501b23236536b8752b1d09f640274e9 100644
--- a/ios/EXDevLauncherController.m
+++ b/ios/EXDevLauncherController.m
@@ -38,7 +38,7 @@
 #define VERSION @ STRINGIZE2(EX_DEV_LAUNCHER_VERSION)
 #endif
 
-static const NSTimeInterval EXDevLauncherDefaultRequestTimeout = 10.0;
+static const NSTimeInterval EXDevLauncherDefaultRequestTimeout = 120.0;
 
 @interface EXDevLauncherController ()
 
